server.rb

```diff
@@ -108,6 +125,20 @@ def add_comment_to_story(story, comment)
   puts "Pivotal Tracker POST #{url}: Response #{response.status}"
 end
 
+def add_label_to_story(story, label)
+  return unless story
+  url = "https://www.pivotaltracker.com/services/v5/projects/#{story.project_id}/stories/#{story.id}/labels"
```

Comment:
Shouldn't the base url be in a constant such that if we change the version in the future we change it in a single place?